### PR TITLE
[R2] Add example on migrating from Cloudflare Images to R2

### PR DESCRIPTION
**Pull request by nora-soderlund**
Added an example of migrating from Cloudflare Images to an R2 bucket via a Worker. Based on my gist @ https://gist.github.com/nora-soderlund/0b30a7edbcf739bc0f6fe421c8d7a6b4.

### CODE REVIEW

**Review by ercanozer — COMMENTED** [review]
Amazing! Thank you for this quick update!

**Comment by KimJ15** [comment]
👋Hi there, we’re going to close this PR due to inactivity

**Comment by DaniFoldi** [comment]
Hi @KimJ15,

Could you tell me why exactly this PR was closed? Inactivity from the PR author, or inactivity from the team?

I've used Nora's gist linked above in the past, and was happy to see her PR that would add it to the docs. Clearly, she spent time on creating and testing the code, and I don't see why a few months passing would render the PR not worthy of merging.

**Comment by dcpena** [comment]
Hi there! I'm going to have this pull get reviewed by a different team member but keep it closed only because this pull request was opened before our docs migration. 

I'll create a new pull request and reference this one as soon as it's ready to keep everyone updated. Thanks!

**Comment by DaniFoldi** [comment]
Hi @dcpena, thanks for the update, that sounds great!

File: content/r2/data-migration/cloudflare-images.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+		const uploadedImages = await exportImagesBatch(env, null);
+
+		return new Response(`${uploadedImages} images was uploaded.`);
+	},
+};
+
+async function exportImagesBatch(env: Env, continuationToken: string | null, imagesCount: number = 0): Promise<number> {
+	const imagesList = await getImagesList(env, continuationToken);
+
+	if(!imagesList.result.images.length) {
```
Comment (harshil1712) [inline content/r2/data-migration/cloudflare-images.md:37]: This causes an infinite loop. Maybe update the condition to check `imageList.result.images.length === imagesCount`?

**Pull request closed without merging**